### PR TITLE
Adjusting wording flow for readability

### PR DESCRIPTION
**Pull request by edneville**
Signed-off-by: ed neville <ed@s5h.net>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,15 @@
 microbit
 ========
 
-_microbit_ contains everything required to get started with the use of Rust to create firmwares for the 
-faboulus [BBC micro:bit][] microcontroller board. This little board
-contains has everything and a kitchensink built-in, even a capable debugging
-interface, so all that one needs to get going with progamming this device is:
+_microbit_ contains everything required to get started with the use of Rust to create firmwares for the faboulus [BBC micro:bit][] microcontroller board. This little board contains everything and has a kitchensink built-in, even a capable debugging interface, so all that one needs to get going with progamming this device is:
```
Comment (therealprof) [inline README.md:4]: Uhm, I "everything and a/the kitchensink" or "everything but the kitchensink" are actually real English phrases. ;)

Comment (edneville) [reply README.md]: Yes, indeed they are, the typo was the prefix "has" so I moved the has to go infront of kitchensink. Although, it should be kitchen sink.

> This little board contains has everything

Comment (therealprof) [reply README.md]: Yeah, but the real fix would be to remove "contains" and keep "has" where it is. I don't want to say it has a kitchen sink built-in. ;)

Comment (edneville) [reply README.md]: I agree, I think that is better.

**Review by therealprof — APPROVED** [review]
LGTM, thanks!

**Pull request merged**